packages/hash/frontend/src/pages/[account-slug]/[page-slug].page.tsx

```diff
@@ -311,34 +307,42 @@ const Page: NextPageWithLayout<PageProps> = ({ blocks }) => {
         </Box>
 
         <Container>
-          <Box display="flex">
+          <Box position="relative">
             <PageIconButton
               accountId={accountId}
               entityId={pageEntityId}
               versionId={versionId}
-              sx={{ mr: 3, alignSelf: "flex-start" }}
               readonly={readonlyMode}
+              sx={({ breakpoints }) => ({
+                mb: 2,
+                [breakpoints.up("lg")]: {
+                  position: "absolute",
+                  top: 0,
+                  right: "calc(100% + 24px)",
+                },
+              })}
```

Comment:
Below this point, when the icon/title are stacked, you want to left align the icon (right now its center aligned) – otherwise the left hand side doesn't line up with the title

Before:

![image](https://user-images.githubusercontent.com/719814/192751011-b96dac22-ea59-4dd7-bb81-833ea88cfb1f.png)

After:

![image](https://user-images.githubusercontent.com/719814/192751025-bd78bdbb-81ab-4fc4-a50a-aad507105903.png)

Reply:
Good catch! Updated here https://github.com/hashintel/hash/pull/1104/commits/d3c8a588715ec84911e685771adf56e35e27f25c

Reply:
@luisbettencourt Yeah, when I checked the [design](https://www.figma.com/file/gydVGka9FjNEg9E2STwhi2/HASH-App?node-id=1503%3A135810) _(internal)_, looks like keeping it centered is intended, and the screenshot Nate posted is also intended on design.

Reply:
I just checked the design. It's left aligned from what I can see. @nonparibus can you check if this needs updating?

Reply:
@nathggns As we can see here, there is `4px` space between the 🚌 emoji and the container. And 🚌 emoji is one of the widest emojis. Imagine ❗️ emoji in this situation. The same spacing would be more than `4px`. So we align the icon button wrapper to the left, and icon is centered inside the button wrapper. Or at least that's how I read this design :D 
Let me know what do you think.

![image](https://user-images.githubusercontent.com/39553853/193726857-b4b16208-de24-48a5-9b96-47e46a48c2cf.png)

Reply:
It appears you're right. I think the design may be wrong. This looks very wrong on the default icon which has a straight left edge as you can see above (but not when using the icon used in the design). @nonparibus can you look?